tech-assess.md

```diff
@@ -0,0 +1,43 @@
+
+New submissions will normally be communicated on email, but if you see one in the list without a reviewer assigned you don’t need to wait to be asked, get on in there! 
+
+##Screening
+
+We’re introducing a step between submission and our full review. We’ll be screening the submission by spending a minimal amount of time (around 15 mins) either reviewing the hosted submission or getting it running locally. In this step we’re looking for:
+
+Has the solution missed any key requirements?
+Does the solution have any unforgivable bugs not acknowledged in the submission email?
+Are the setup instructions insufficient to get it running locally?
+
+If any of these are true, we’ll decide either not to take the submission further or to ask the candidate to try again or provide more information. Record this decision in the Developer Assessment Google sheet and notify Stef, who will then contact the candidate.
+
+##Review
+
+In the Google Sheet you will find the following columns to complete. Some of them we’d also like you to colour green, amber or red depending on whether the criteria have been fully, partially or not met. 
+
+**Quality (R/A/G)** - Compare the solution to the requirements laid out in the technical challenge. If they have met the requirements fully you only need to note this. Anything the candidate has missed, please note here. 
+
+**Structure (R/A/G)** - Evaluate the architecture choices, code quality and robustness of the application. Reference previous assessments for ideas on what to look out for. In short, ask yourself how easy would it be for someone else (assuming knowledge of the framework) to pick up and extend the application. 
+
+**UX/UI (R/A/G)** - We’re not recruiting designers with this challenge. However, we do expect our engineers to know what a good website looks like and to have the basic skills necessary to achieve one. If you find usability issues that could have been addressed using simple CSS and some attention to detail, you should note these and consider whether this criteria has been met.
+
+**Bonus points (R/A/G)** - Quality, structure and UX/UI are not catch-all and a candidate may find a way to impress outside of these criteria, so note those things under bonus points. We’re very interested in finding candidates who do go above and beyond, but also understand that some may not have the time to dedicate to the challenge. If you are unable to find any bonus points, please do mark the cell red. The main focus of deciding to take a candidate further will be on the criteria above.
+
+**Approximate skill level** - It’s difficult to judge on one assessment, and we’ll take your judgement with a pinch of salt, but try to estimate where the candidate would sit in terms of skill level within our team. 
```

Comment:
Perhaps this could read
```
It’s difficult to judge on one technical assessment, and we’ll take your judgement in the context of other interviews
```